ansible/enable_chn.yml

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ansible-playbook
+# Copyright 2020-2021 Hewlett Packard Enterprise Development LP
```

Comment:
Need 2022 - format is incorrect, but looks like the license checker is skipping this file...